phpBB/install/app.php

```diff
@@ -20,9 +20,9 @@
 $phpbb_root_path = '../';
 $phpEx = substr(strrchr(__FILE__, '.'), 1);
 
-if (version_compare(PHP_VERSION, '5.4') < 0)
+if ( version_compare(PHP_VERSION, '5.4.7', '<') || version_compare(PHP_VERSION, '7.3-dev', '>=') )
```

Comment:
Should be this according to coding guidelines:
```suggestion
if (version_compare(PHP_VERSION, '5.4.7', '<') || version_compare(PHP_VERSION, '7.3-dev', '>='))
```

Reply:
Ok, though I think that the code's readability gains here.